app/components/CategoryHeader.android.js

```diff
@@ -10,13 +11,33 @@ const styles = StyleSheet.create({
   },
 });
 
-const CategoryHeader = ({ category }) => (
-  <ToolbarAndroid
-    title={category.name}
-    style={styles.toolbar}
-    titleColor="white"
-  />
-);
+class CategoryHeader extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      icon: null
+    }
+  }
+
+  componentWillMount() {
+    Icon.getImageSource('bars', 22, 'white').then((source) => this.setState({ icon: source }));
```

Comment:
i checked example project and they don't clean it up
https://github.com/oblador/react-native-vector-icons#usage-as-png-imagesource-object
so i hope it's fine